tensorflow/python/keras/applications/imagenet_utils.py

```diff
@@ -414,10 +415,10 @@ def validate_activation(classifier_activation, weights):
     return
 
   classifier_activation = activations.get(classifier_activation)
-  if classifier_activation not in [
+  if classifier_activation not in {
```

Comment:
Why do you want to change this to a dict?

Reply:
This was just a very minor edit made based on the fact that the `in` operator works faster on sets (not dict) than lists. I can write a `timeit` test if you want, but this is already pretty well-documented on this [stackoverflow post](https://stackoverflow.com/questions/2831212/python-sets-vs-lists). Thanks!

Reply:
Thanks for the note! We don't need tests for this.